tests/unit_tests/cli/test_integration_cli.py

```diff
@@ -447,7 +447,7 @@ def test_that_prior_is_not_overwritten_in_ensemble_experiment(
     ],
 )
 @pytest.mark.usefixtures("copy_poly_case")
-def test_that_the_cli_raises_exceptions_when_parameters_are_missing(mode):
+def test_that_the_cli_raises_exceptions_when_parameters_are_missing(tmpdir, mode):
```

Comment:
Is the added argument `tmpdir` used in the test?

Reply:
No. It's a stray. I'll remove that one. Thanks!